data-model/src/main/scala/za/co/absa/enceladus/model/properties/propertyType/package.scala

```diff
@@ -37,7 +37,11 @@ package object propertyType {
     override def isValueConforming(value: String): Try[Unit] = Success(Unit)
   }
 
-  case class EnumPropertyType(allowedValues: Set[String], suggestedValue: Option[String]) extends PropertyType {
+  case class EnumPropertyType(allowedValues: Seq[String], suggestedValue: Option[String]) extends PropertyType {
+    require(allowedValues.nonEmpty, "At least one allowed value must be defined for EnumPropertyType.")
```

Comment:
Like these wordings. And was not aware/forgot about the `require` usage. 👍

Reply:
Thanks